apps/prairielearn/src/lib/features/index.ts

```diff
@@ -1,5 +1,8 @@
 import { FeatureManager, FeatureGrantType } from './manager';
 
-const features = new FeatureManager([]);
+const features = new FeatureManager([
+  // Can only be applied to courses/institutions.
+  'process-questions-in-worker',
+]);
```

Comment:
🎉 first feature flag!

We'll see how well inline documentation of what this flag can be granted to holds up relative to enforcing this programmatically.